run_local_with_dependencies.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-sm --start API_DEFINITION THIRD_PARTY_APPLICATION API_EXAMPLE_MICROSERVICE API_PLATFORM_MICROSERVICE -f
-sm --start ASSETS_FRONTEND -r 3.11.0 -f
+sm --start API_DEFINITION THIRD_PARTY_APPLICATION API_EXAMPLE_MICROSERVICE API_PLATFORM_MICROSERVICE -r
```

Comment:
What does '-r' do when it doesn't have any version specified? What would be the difference with just not having -r at all?

Reply:
Good point, nothing I guess. I remember being told to use `-r` instead of `-f` when everything moved to being run from RELEASE. I'll chase it up just to make sure.

Reply:
> It depends on the service-manager config for the service. AFAIK most services have config to say ‘always use release’, so that flag wouldn’t change anything when running one of those.
I've checked service-manager-config for these services and they all have `"always_run_from": "RELEASE"` so I'll remove this `-r` flag.